swagger_server/initdb.py

```diff
@@ -0,0 +1,77 @@
+from concurrent.futures.process import ProcessPoolExecutor
+from concurrent.futures.thread import ThreadPoolExecutor
+from random import sample, randrange
+
+from neomodel import config
+
+from swagger_server.orm.DistanceORM import SampleNode, LineageNode
+
+config.DATABASE_URL = 'bolt://neo4j:@127.0.0.1:7687'
+config.ENCRYPTED_CONNECTION = False
+
+
+N_LEAVES = 2042
```

Comment:
This should be `2043`, as we want leaf nodes from `leaf_0000` to `leaf_2042` inclusive.

Reply:
Fixed